definitions/tools/vcf_readcount_annotator.cwl

```diff
@@ -7,7 +7,7 @@ label: "add bam_readcount info to vcf"
 baseCommand: ["vcf-readcount-annotator"]
 requirements:
     - class: DockerRequirement
-      dockerPull: mgibio/cle
+      dockerPull: mgibio/vcf_annotation_tools-cwl
```

Comment:
If we're switching to smaller containers can we start to version this?

Reply:
Done.